components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -325,6 +325,15 @@ protected boolean handlePostAuthentication(Event event, String userName, UserSto
             }
         }
 
+        // Skip account lock if account lock by pass is enabled for the userstore manager.
+        if (AccountUtil.isAccountLockByPassForUserStore(userStoreManager)) {
```

Comment:
Don't we need to move this check to [1]? We only need to do this if the authentication failed right?

[1] - https://github.com/wso2-extensions/identity-event-handler-account-lock/pull/103/files#diff-be95aff6d433d7b6c2456b282fd5f8693f3fae06f93a9cd4a8b84ec1d596264eR393

Reply:
Hence account lock is by passed for the userstore thought of putting this check here.

Reply:
Addressed with https://github.com/mevan-karu/identity-event-handler-account-lock-1/pull/103/commits/32c5c8ca9ea8e9fe983b87531c2ee9cf864f23f1